script/load_templates

```diff
@@ -224,7 +224,7 @@ sub post_entry {
         }
     }
 
-    my $res = $client->post($table_url, form => \%param)->res;
+    my $res = $client->post($table_url->query(%param))->res;
```

Comment:
I found out why it didn't work with form params.

`$table_url` is used before with a get request:

`my $res = $client->get($table_url, form => \%param)->res;`

So the query parameters survive when doing the post request, and so every parameter is passed in the QUERY string and as form parameters, so they get duplicated and are interpreted as an array.

The fix is to just do

`$table_url = $url->clone->path($options{'apibase'} . '/' . decamelize($table));`
here again.

Reply:
Oh, and in the get request above it should be `->query` instead of `form =>`, but I guess that's done automatically behind the scenes.

Reply:
So the `get` ends up modifying the URL behind the scenes... nice....